apps/docs/content/components/Image.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: Image
+status: Experimental
+source: https://github.com/primer/brand/blob/main/packages/react/src/Image/Image.tsx
+storybook: '/brand/storybook/?path=/docs/components-image--playground'
+description: Use Image to create both normal and responsive image elements.
+---
+
+import {Label} from '@primer/react'
+
+```js
+import {Image} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+This component uses the `img` element by default.
+
+```jsx live
+<Image
+  src="https://via.placeholder.com/600x400/d3d9df/d3d9df.png"
+  alt="placeholder, blank area with an off-white background color"
+/>
+```
+
+### Picture
+
+The `as` prop can be used to set the container of the image to be use `picture`.
+
+```jsx live
+<Image
+  as="picture"
```

Comment:
This example is setting an empty `source`. Could we add a test to guard this please? Should only appear if a `source` is defined.

![Screenshot 2023-04-18 at 14 32 23](https://user-images.githubusercontent.com/13340707/232793425-a23d64d2-fcb4-4ed6-811d-e7ccfb7c72e4.png)

I assume most users of `as="picture"` would want to set some sources manually. Could we update this example to show how you would do this? I'm not quite clear on that myself right now either.

Reply:
This would require access to children if we want to do that simply. I can include that but it would overcomplicate the component a bit possibly.

Reply:
I think we should try to avoid doing this because it is going to complicate things quite a bit possibly.

Reply:
This has been updated to work when defined but not a custom setting.

Reply:
This has be discussed and resolved here: https://github.com/primer/brand/pull/229#discussion_r1171768591